src/ServiceInsight/App_Packages/Particular.Licensing/License.cs

```diff
@@ -41,7 +41,7 @@ public License()
 
         public bool ValidForApplication(string applicationName)
         {
-            return ValidApplications.Contains(applicationName) || ValidApplications.Contains("All");
+            return ValidApplications.Contains(applicationName) || ValidApplications.Contains("All") || applicationName == "NServiceBus";
```

Comment:
Discussed in @Particular/serviceinsight-maintainers sync today. We're OK with this. It doesn't affect SI.